src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -304,18 +299,18 @@ public class CoinstakeContext
         private CancellationTokenSource stakeCancellationTokenSource;
 
         /// <summary>Provider of IBD state.</summary>
-        private IInitialBlockDownloadState initialBlockDownloadState;
+        private readonly IInitialBlockDownloadState initialBlockDownloadState;
 
         /// <summary>State of time synchronization feature that stores collected data samples.</summary>
         private readonly ITimeSyncBehaviorState timeSyncBehaviorState;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="PosMinting"/> class.
         /// </summary>
+        /// <param name="blockProvider">Block provider.</param>
```

Comment:
@dangershony in the end, what have we decided ? are we going to update the commenting policy for cases like that where comments are just noise, or is it still a hotly debated topic ?

Reply:
@dangershony well we either remove all parameter comments or keep
them up-to-date. Doesn’t make sense to have half-completed comments.

Reply:
@fenix2222 please don't take my question as personal attack, it is just a side question :) I just find it sad to have to write this kind of things because of a rule. I don't mind if people do it, but I personally can't understand why I would be forced to do that.
To disable warnings: https://stackoverflow.com/questions/7982525/visual-studio-disabling-missing-xml-comment-warning